modules/get-started/pages/install-beta.adoc

```diff
@@ -0,0 +1,154 @@
+= Install 23.3 Beta
+:description: Learn how to install the Beta version. 
+
+Redpanda Beta versions provide users the opportunity to test and share feedback on new features before they're finalized in general availability. Beta versions, like `v23.3.1-rc1`, are published to `redpanda-unstable` as release candidate (RC) builds. RC builds are not recommended for production use.
+
+To install the Beta version, select your environment.
+
+[tabs]
+=====
+Docker::
++
+--
+
+. Change your image repository from:
++
+```
+image: docker.redpanda.com/redpandadata/redpanda
+```
++
+to:
++
+```
+image: docker.redpanda.com/redpandadata/redpanda-unstable
+```
+
+. Pull the 23.3 RC build from https://hub.docker.com/r/redpandadata/redpanda-unstable/tags[dockerhub^]. For example:
++
+```
+docker pull docker.redpanda.com/redpandadata/redpanda-unstable:v23.1.1-rc1
+```
+
+xref:./quick-start.adoc[Redpanda Quickstart]: You can test Beta features with the quickstart by replacing the image in your `docker-compose.yml` file with the RC build. For example, the following `docker-compose.yml` includes the RC build for a single broker quickstart.
+
+.Reveal the YAML content
+[%collapsible]
+====
+```
+version: "3.7"
+name: redpanda-quickstart
+networks:
+  redpanda_network:
+    driver: bridge
+volumes:
+  redpanda-0: null
+services:
+  redpanda-0:
+    command:
+      - redpanda
+      - start
+      - --kafka-addr internal://0.0.0.0:9092,external://0.0.0.0:19092
+      # Address the broker advertises to clients that connect to the Kafka API.
+      # Use the internal addresses to connect to the Redpanda brokers'
+      # from inside the same Docker network.
+      # Use the external addresses to connect to the Redpanda brokers'
+      # from outside the Docker network.
+      - --advertise-kafka-addr internal://redpanda-0:9092,external://localhost:19092
+      - --pandaproxy-addr internal://0.0.0.0:8082,external://0.0.0.0:18082
+      # Address the broker advertises to clients that connect to the HTTP Proxy.
+      - --advertise-pandaproxy-addr internal://redpanda-0:8082,external://localhost:18082
+      - --schema-registry-addr internal://0.0.0.0:8081,external://0.0.0.0:18081
+      # Redpanda brokers use the RPC API to communicate with eachother internally.
+      - --rpc-addr redpanda-0:33145
+      - --advertise-rpc-addr redpanda-0:33145
+      # Tells Seastar (the framework Redpanda uses under the hood) to use 1 core on the system.
+      - --smp 1
+      # The amount of memory to make available to Redpanda.
+      - --memory 1G
+      # Mode dev-container uses well-known configuration properties for development in containers.
+      - --mode dev-container
+      # enable logs for debugging.
+      - --default-log-level=debug
+	image: docker.redpanda.com/redpandadata/redpanda-unstable:v23.3.1-rc1
```

Comment:
Suggestion: I want to say I've seen this in our docs but don't recall where exactly, do we have the ability to highlight specific lines in a code block? It might help to just quickly highlight to the reader that this is the specific `image` line that they need to change

Reply:
https://deploy-preview-136--redpanda-docs-preview.netlify.app/23.3/get-started/install-beta/?tab=tabs-1-docker#tabs-1-docker

Reply:
Cool, today I learned! Thanks @micheleRP